engine/src/mac-dialog.mm

```diff
@@ -646,12 +646,6 @@ void MCPlatformBeginFileDialog(MCPlatformFileDialogKind p_kind, MCPlatformWindow
 	else 
 		[t_panel setCanCreateDirectories: YES];
 	
-	if ([t_panel respondsToSelector:@selector(isAccessoryViewDisclosed)])
```

Comment:
Hi Panos, I tested this on El Capitan, where the bug was originally reported - removing this code doesn't appear to have any negative effect on the appearance of the accessoryview so i considered it safe to remove.